runtime/compiler/optimizer/J9ValuePropagation.cpp

```diff
@@ -1206,34 +1207,34 @@ J9::ValuePropagation::constrainRecognizedMethod(TR::Node *node)
 
             if (isStoreFlattenableArrayElement && !owningMethodDoesNotContainStoreChecks(this, node))
```

Comment:
I think you intended to remove this test of `owningMethodDoesNotContainStoreChecks`, as you've added a new test of that below.  However, see also my comment above about whether `owningMethodDoesNotContainStoreChecks` should suppress both `ArrayStoreCHK` and `NULLCHK` (which was what was originally intended) or if it should be split into two separate tests:  `owningMethodDoesNotContainStoreChecks` and `owningMethodDoesNotContainNullChecksForArrayStores`, say.

Reply:
>I think you intended to remove this test of owningMethodDoesNotContainStoreChecks, as you've added a new test of that below.

Whoops, sorry my bad. Just realized I forgot to remove it

Reply:
Addressed in c4346cc